style/scss/_team.scss

```diff
@@ -33,35 +34,101 @@ $tablet-line-height-p: 2rem;
     margin-bottom: -0.5rem;
   }
 
+  p {
+    font-size: 1.6rem;
+  }
+
   .team-line {
     width: 23rem;
   }
 
-  p {
+  &__description {
     width: auto;
     font-size: 1.6rem;
     line-height: 130%;
     text-align: center;
+    text-wrap: balance; // Wrap text evenly
+  }
+
+  &__members-wrapper {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 3rem;
+
+    #team-navbar {
+      position: sticky;
+      top: 8.5rem;
+      width: 100%;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      flex-wrap: wrap;
+      gap: 0.75rem;
+      z-index: 10;
+
+      .team-nav-btn {
+        font-size: 1.6rem;
+        font-weight: 600;
+        letter-spacing: 1px;
```

Comment:
Please remove this letter spacing, it should be consistent across all buttons (0px)

Reply:
Removed letter spacing to be consisntent with all the buttons on the website